prisma/schema.prisma

```diff
@@ -21,6 +21,16 @@ model User {
 model Activity {
   id    Int     @id @default(autoincrement())
   name String
+  amount Int
+  unit String
   timesDone  Int @default(0)
   userId Int
 }
+
+model ActivityRecord {
+  id    Int     @id @default(autoincrement())
+  userId      Int
+  activityId   Int
+  addedAmount Int
+  date         DateTime @default(now())
```

Comment:
It might be better to add some context with some better naming. createdAt, updatedAt or something along those lines